### PR TITLE
Added a "smoke test" for the application URLs and moved one test location

### LINKED ISSUE #18: Add tests
According to the best practices, each page should at least have a smoke test. Currently, there is only one page with a test

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #18

### CODE REVIEW

File: src/AppBundle/Tests/Controller/BlogControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+/**
+ * Functional test for the controllers defined inside BlogController.
+ * See http://symfony.com/doc/current/book/testing.html#functional-tests
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ * $ cd your-symfony-project/
+ * $ phpunit -c app
+ *
+ * @author Ryan Weaver <weaverryan@gmail.com>
```
Comment (WouterJ) [inline src/AppBundle/Tests/Controller/BlogControllerTest.php]: only you are the author?

Btw, Symfony core never adds authorship to tests

File: src/AppBundle/Tests/Controller/BlogControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use AppBundle\Entity\Post;
+
+/**
+ * Functional test for the controllers defined inside BlogController.
+ * See http://symfony.com/doc/current/book/testing.html#functional-tests
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ * $ cd your-symfony-project/
+ * $ phpunit -c app
```
Comment (WouterJ) [inline src/AppBundle/Tests/Controller/BlogControllerTest.php]: please indent them (and I prefer an empty after the text)

File: src/AppBundle/Tests/Controller/BlogControllerTest.php (hunk 1 above)
Comment (weaverryan) [inline src/AppBundle/Tests/Controller/BlogControllerTest.php]: I'm happy to be thought of at least :)

File: src/AppBundle/Tests/Controller/BlogControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+class BlogControllerTest extends WebTestCase
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+        $crawler = $client->request('GET', '/blog/');
+
+        $this->assertEquals(Post::NUM_ITEMS, $crawler->filter('article.post')->count(),
+            'The homepage displayes the right number of posts'
+        );
```
Comment (WouterJ) [inline src/AppBundle/Tests/Controller/BlogControllerTest.php]: You should use `assertCount` and it is not proper PSR indenting. Please use

```php
$this->assertCount(
    Post::NUM_ITEMS,
    $crawler->filter('article.post'),
    'The homepage displays the right number of posts'
);
```

Comment (WouterJ) [inline src/AppBundle/Tests/Controller/BlogControllerTest.php]: Btw, there are other places in the demo app which aren't line-wrapped. I think we should be consistent with it

**Comment by javiereguiluz** [comment]
@WouterJ I've implemented all your suggestions. Thanks for the code review.

**Comment by WouterJ** [comment]
:+1: Thanks for your quick reactions!

**Pull request closed without merging**